.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
         "@typescript-eslint"
     ],
     rules: {
-        semi: "error"
+        semi: "error",
+        "@typescript-eslint/no-explicit-any": "off"
```

Comment:
We should still discourage the use, hence warning:
```suggestion
        "@typescript-eslint/no-explicit-any": "warn"
```

Reply:
Oops. Misunderstood.